CRM/Utils/Money.php

```diff
@@ -61,6 +61,13 @@ public static function format($amount, $currency = NULL, $format = NULL, $onlyNu
       $valueFormat = $config->moneyvalueformat;
     }
 
+    if (!empty($valueFormat) && $valueFormat !== '%!i') {
+      Civi::log()->warning(
```

Comment:
This is the same as CRM_Core_Error::deprecatedWarning - but maybe use that for grepability

Reply:
@eileenmcnaughton there is deprecatedFunctionWarning but that assumes you are deprecating calls to a specific function which we are not doing here and there is no deprecatedWarning function in CRM_Core_Error

Reply:
yeah - but we've been using it everywhere.... regardless of whether it's a function

Reply:
@eileenmcnaughton we have also been using this pattern as well

```
Civi/Angular/Manager.php:            'civi.tag' => 'deprecated',
Civi/Payment/PropertyBag.php:      Civi::log()->warning($message, ['civi.tag' => 'deprecated']);
CRM/Utils/Geocode.php:      ['civi.tag' => 'deprecated']
CRM/Price/BAO/LineItem.php:              Civi::log()->warning('Per https://lab.civicrm.org/dev/core/issues/15 this data fix should not be required. Please log a ticket at https://lab.civicrm.org/dev/core with steps to get this.', ['civi.tag' => 'deprecated']);
CRM/Contribute/BAO/Contribution.php:        ->warning('Deprecated bit of code, please log a ticket explaining how you got here!', ['civi.tag' => 'deprecated']);
CRM/Dedupe/Merger.php:        ->warning("Deprecated hook ::merge in context of 'cidRefs. Use entityTypes instead.", ['civi.tag' => 'deprecated']);
CRM/Member/BAO/Query.php:          ['civi.tag' => 'deprecated']
CRM/Member/BAO/Query.php:          ['civi.tag' => 'deprecated']
CRM/Member/BAO/Query.php:          ['civi.tag' => 'deprecated']
CRM/Core/Payment/AuthorizeNetIPN.php:        Civi::log()->warning('Payment Processor does not match the recurring processor id.', ['civi.tag' => 'deprecated']);
CRM/Core/BAO/Cache/Psr16.php:          ['civi.tag' => 'deprecated']
CRM/Core/BAO/Cache/Psr16.php:          ['civi.tag' => 'deprecated']
CRM/Core/BAO/Cache/Psr16.php:          ['civi.tag' => 'deprecated']
CRM/Core/BAO/Website.php:      \Civi::log()->warning(ts('Calling website:create with vars other than $params is deprecated. Use process'), ['civi.tag' => 'deprecated']);
CRM/Core/BAO/Website.php:        \Civi::log()->warning(ts('Calling website:create for multiple websites $params is deprecated. Use process'), ['civi.tag' => 'deprecated']);
CRM/Core/BAO/PrevNextCache.php:      Civi::log()->warning('Deprecated code path. Values should not be set this is going away in the future in favour of specific function params for each column.', array('civi.tag' => 'deprecated'));
CRM/Core/BAO/Setting.php:        Civi::log()->warning('Deprecated Path: There is a setting (' . $settingField . ') not correctly defined. You may see unpredictability due to this. CRM_Core_Setting::setItems', ['civi.tag' => 'deprecated']);
CRM/Mailing/BAO/Mailing.php:      \Civi::log('Parameter $ids is no longer used by Mailing::add. Use the api or just pass $params', ['civi.tag' => 'deprecated']);
CRM/Mailing/BAO/Mailing.php:      \Civi::log('Parameter $ids is no longer used by Mailing::create. Use the api or just pass $params', ['civi.tag' => 'deprecated']);
CRM/Mailing/BAO/Mailing.php:    \Civi::log('This function is deprecated, use CRM_Mailing_BAO_MailingJob::del instead', ['civi.tag' => 'deprecated']);
```